.github/workflows/composite/dp-integ-tests/action.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: dp-integ-tests
+description: Run DP integration tests
+
+inputs:
+  make_tests_target:
+    required: true
+    description: Make target for running relevant tests
+  test_dir:
+    required: true
+    description: Directory to store tests results
+    default: /tmp/integration-tests-results
+  github_token:
+    required: true
+    description: Github token
```

Comment:
The Github token does not need to be handed in. It can be accessed via `${{ github.token }}`.

Reply:
Thanks. I didn't know that